docs/testrunner-toolkit/faqs.md

```diff
@@ -81,3 +81,8 @@ $> export HTTP_PROXY=http://my.proxy.org:3128/
 $> export HTTPS_PROXY=http://my.proxy.org:3128/
 $> saucectl run -e HTTP_PROXY=${HTTP_PROXY} -e HTTPS_PROXY=${HTTPS_PROXY} --test-env docker
 ```
+
+## What is the maximum execution time of TestCafe tests ?
+
+The maximum execution time of TestCafe tests is limited to 30 minutes. 
```

Comment:
Hey @valentyn88  -- This seems fine to me -- is it specific enough? I am not a developer, so forgive me if this is super obvious, but do we need to provide any additional details about:
- what will happen if a test exceeds this limit (test fails?)
- how to make sure tests _don't_ exceed this limit?
- why this limit exists?
Thanks!

Reply:
Hey @sweeneyskirt-sl 

- If a test exceeds a limit test-runner stops execution and uploads assets (videos, screenshots, logs, etc..) to our platform. 
- To make sure tests don't exceed this limit users should split their tests and not wait 30 mins. 
- If a test runner gets stuck we need to have a way to stop a VM. We have another service that can stop a VM after 3 h 30 mins, but it's too long.